src/spartan/sumcheck.rs

```diff
@@ -292,7 +337,7 @@ impl<G: Group> SumcheckProof<G> {
 // ax^3 + bx^2 + cx + d stored as vec![a,b,c,d]
 #[derive(Debug)]
 pub struct UniPoly<G: Group> {
-  coeffs: Vec<G::Scalar>,
+  pub coeffs: Vec<G::Scalar>,
```

Comment:
I'd add a getter. But would not open this as `pub` as end users might want to touch coeffs inside when they shouldn't.

Reply:
Good point, agree. Added this change in the next commit among other updates related to the comments.